### ISSUE TITLE
[Question] Is there a way to format Cucumber's DataTable ?

### ISSUE THREAD

**Issue by jordanviking** [Alterhu/CucumberPeople#11]
Is there a shortcut (ike Ctrl+Shift+F) to format the feature files and it´s DataTables?

For example, format this DataTable
```
 And I check if the ssh command returned
    | result | error_message |
    | Execution result: 0 | Error running the script |
    | Restoring Predictive Billing info for contract:| Error in restore |
```

Like this?
```
And I check if the ssh command returned
    | result                                          | error_message            |
    | Execution result: 0                             | Error running the script |
    | Restoring Predictive Billing info for contract: | Error in restore         |
```
I tried to install the Aptana´s Cucumber Bundle, but nothing happened.

**Comment by Alterhu** [comment]
yes, You can take to use the shortcut CTRL + SHIFT + F to format the cucumber file for datatable ,but maybe the formatting have some limitation ,please let me know the original data table you want to format?

**Comment by jordanviking** [comment]
I tried to format a simple feature file:
...
#encode UTF8
Feature: Create client

@shell 
Scenario: Run shell commands 
	Given I login on my system
	Then I create a client with params:
|name|las_name|address|city|
|Jordan|Herbert|%th Avenue|New York|
...

The DataTable's indentation is ok, but the pipes aren´t aligned, as the picture shows:
![image](https://cloud.githubusercontent.com/assets/1587495/8312717/c8845166-19b4-11e5-8be5-e0a00e74de78.png)

**Comment by Alterhu** [comment]
Yes, some limitation for this , and hope I can fix it in future version,thanks for pointing it out .
Will let you know if i can fix it :)

**Comment by jordanviking** [comment]
Aptana´s cucumber ruble have the table align feature, but I can´t install it, maybe you can reuse their code:
https://github.com/aptana/cucumber.ruble/blob/stable/lib/cucumber/mate/table_aligner.rb

Thanks a lot! Keep doing the nice work! I´ll try to help as much as I can!
